src/wallet/wallet.cpp

```diff
@@ -1408,13 +1407,25 @@ bool CWallet::IsHDEnabled()
     return !hdChain.masterKeyID.IsNull();
 }
 
+void CWallet::SetWalletFlag(uint64_t flag)
+{
+    walletFlags |= flag;
```

Comment:
I think this method should be plural, since there is no distinction between a single flag and multiple flags in its operation and it might be useful to be able to do the latter.

Then, of course, you have confusion. So, a verb change in the method name might be useful.